src/components/Artists.js

```diff
@@ -0,0 +1,95 @@
+import React, { Component } from "react";
+import styled from "styled-components/macro";
+
+import Grid from "@material-ui/core/Grid";
+import Card from "@material-ui/core/Card";
+import CardActionArea from "@material-ui/core/CardActionArea";
+import CardMedia from "@material-ui/core/CardMedia";
+import Typography from "@material-ui/core/Typography";
+
+const ARTISTS_INFO = [
+  {
+    imageUrl:
+      "https://consequenceofsound.files.wordpress.com/2010/10/2010_10oct_24_jayzhiphopadvice.jpg?quality=80",
+    artistName: "Jay Z"
+  },
+  {
+    imageUrl: "https://lyricsforme.com/upload/image/artist/adele-profile.jpg",
+    artistName: "Adele"
+  },
+  {
+    imageUrl:
+      "http://static1.squarespace.com/static/578f89682994ca6eb42dac86/578f8b86d1758e8ac840fdb2/5b6c7fc68a922d83a728c122/1533839950211/LUDACRIS.jpg?format=1000w",
+    artistName: "Ludacris"
+  },
+  {
+    imageUrl:
+      "https://images.genius.com/fce1e2b08952b1596484d7ea9ce00db2.1000x1000x1.jpg",
+    artistName: "Xzibit"
+  },
+  {
+    imageUrl:
+      "https://fanart.tv/fanart/music/1d11e2a1-4531-4d61-a8c7-7b5c6a608fd2/artistthumb/ice-cube-53ffd45ce495f.jpg",
+    artistName: "Ice Cube"
+  },
+  {
+    imageUrl:
+      "https://www.shropshirestar.com/resizer/hqU1_vsIGbZkvKdwRSzpDYf59Vo=/1000x0/filters:quality(100)/arc-anglerfish-arc2-prod-shropshirestar-mna.s3.amazonaws.com/public/RBFEKBZ3PZDPHM662ZRIJZJNTA.jpg",
+    artistName: "Britney Spears"
+  }
+];
+
+const ArtistsContainer = styled(Grid)`
+  display: flex;
+  flex-wrap: wrap;
+  width: calc(100% - ${props => props.spacing}px);
+  margin: 0 auto;
+`;
+const ArtistContainer = styled(Grid)``;
```

Comment:
please do not wrap component into `styled` if you do not add styles to it

Reply:
I can not do this because styles are inherited from the component specified in the properties. Either call the Grid component, or use const ArtistContainer = styled (Grid) ``;